### DISCUSSION TITLE
Linux Database Location

### DISCUSSION THREAD

**Discussion by furkantiv** [tekartik/sqflite#689] [Q&A] [answered]
Where is the database located on linux?

**Comment by alextekartik** [comment, marked as answer]
```dart
var factory = databaseFactoryFfi;
print(await factory.getDatabasesPath());
```

should give you something like:

```
<current_dir>/.dart_tool/sqflite_common_ffi/databases
```

which is just a placeholder location. The recommendation for Linux (and Desktop in General) is to use `path_provider` package to find the best location (I'm not aware of any recommendation on Linux but there might be common locations) and use an absolute path when opening a database.

**Reply by furkantiv** [reply]
Thank you for answer
